js/popups/tablemenupopup.ts

```diff
@@ -147,6 +152,7 @@ export class TableMenuPopup {
         this.handHistoryAllowed(playerIsInGame && currentTable.lastHandHistory() != null);
         this.leaveAllowed(myPlayer != null && !currentTable.myPlayerInGame() && myPlayer.IsSitoutStatus());
         this.accountStatusAllowed(this.authInformation.authenticated());
+        this.skipDeals(currentTable.isSitOut());
```

Comment:
nitpicking: Could it be `currentTable.skipDeals()` for consistency?

Reply:
we don't have skipDeals on tableview